internal/bibimg/imagetypes.go

```diff
@@ -0,0 +1,104 @@
+package bibimg
+
+import (
+	"fmt"
+	"slices"
+	"sort"
+	"strings"
+)
+
+type imageType struct {
+	Export string
+	ISO    bool
+	Legacy bool
+}
+
+var supportedImageTypes = map[string]imageType{
+	// XXX: ideally we would look how to consolidate all
+	// knownledge about disk based image types into the images
```

Comment:
```suggestion
	// knowledge about disk based image types into the images
```
very important suggestion